charts/k8s-infra/values.yaml

```diff
@@ -131,6 +137,12 @@ presets:
         id: extract_metadata_from_filepath
         regex: '^.*\/(?P<namespace>[^_]+)_(?P<pod_name>[^_]+)_(?P<uid>[a-f0-9\-]+)\/(?P<container_name>[^\._]+)\/(?P<restart_count>\d+)\.log$'
         parse_from: attributes["log.file.path"]
+        output: add_cluster_name
+      # Add cluster name attribute from environment variable
+      - id: add_cluster_name
```

Comment:
@nityanandagohain why is this chaining required? I see a case for initial runtime detection but is there a reason to chain this for attribute extraction also?